cmake/thirdparty/SetupSeracThirdParty.cmake

```diff
@@ -482,19 +484,6 @@ if (NOT SERAC_THIRD_PARTY_LIBRARIES_FOUND)
         set(ENABLE_FORTRAN ON CACHE BOOL "" FORCE)
     endif()
 
```

Comment:
Apparently we needed to find PETsc several times lol.... Good catch @chapman39 !